cirrus-ci_artifacts/test/run_all_tests.sh

```diff
@@ -1,21 +1,13 @@
 #!/bin/bash
 
-if [[ "$CIRRUS_CI" != "true" ]]; then
-    echo -e "\nSkipping: Test must be executed under Cirrus-CI\n"
-    exit 0
-fi
-
 TESTDIR=$(dirname ${BASH_SOURCE[0]})
-
-cd "$TESTDIR/../"
-virtualenv testvenv
+cd "$TESTDIR"
 
 set -a
+virtualenv testvenv
 source testvenv/bin/activate
-set +a
-
 testvenv/bin/python -m pip install --upgrade pip
-pip3 install --requirement ./requirements.txt
+pip3 install --requirement ../requirements.txt
```

Comment:
Pulling from pypi vs. the distribution repositories could lead to different versions of the python libraries being used.

Reply:
\*sigh\* last I checked (a year ago) Fedora doesn't provide `gql` or any other GraphQL client, only a server :cry:

I think there's a way to fix the requirements onto specific versions, but I can't remember the syntax...

Reply:
...right, it's [the `~=` compatibility specifier.](https://peps.python.org/pep-0440/#version-specifiers)